provisioning-workflows/zOS/NetworkResources/Deprovision_Network_Resource.xml

```diff
@@ -0,0 +1,182 @@
+        <description>Return allocated network resources back to Config Assistant</description>
+        <step name="returnDVIPA" optional="false">
+            <title>Return DVIPA</title>
+            <description>Invoke RESTful API to return IP address back to pool.</description>
+            <condition>
+                <expression>"1" == "1"</expression>
+                <description>Check that previous step has completed</description>
+                <targetStateSet>
+                    <description>Set state to Skipped if IP_ADDRESS_ID is not set</description>
+                    <extendStateExpression>
+                        <description>Skip if IP_ADDRESS_ID is null</description>
+                        <expression>${instance_IP_ADDRESS_ID} == null</expression>
+                        <targetState>Skipped</targetState>
+                    </extendStateExpression>
+                     <extendStateExpression>
+                        <description>Skipped if IP_ADDRESS_ID is blank</description>
+                        <expression> ${instance_IP_ADDRESS_ID} == ""</expression>
+                        <targetState>Skipped</targetState>
+                    </extendStateExpression>
+                </targetStateSet>
+            </condition>
+            <instructions substitution="false">
+            Execute step to return the IP address back to z/OS Communications Server.
+            </instructions>
+            <weight>10</weight>
+            <skills>REST</skills>
+            <autoEnable>true</autoEnable>
+            <canMarkAsFailed>false</canMarkAsFailed>
+            <rest>
+                <httpMethod>POST</httpMethod>
+                <uriPath substitution="true">/zosmf/resource-mgmt/rest/1.0/rdp/network/ip/actions/release</uriPath>
+                <requestBody substitution="true">
+                    {
+                    "template-uuid":"${_workflow-templateID}",
+                    "template-name":"${_workflow-templateName}",
+                    "tenant-id":"${_workflow-tenantID}",
+                    "network-parms" :
+                            {
+                            "ip-id":"${instance-IP_ADDRESS_ID}"
+                            }
+                    }
+                </requestBody>
+                <expectedStatusCode>204</expectedStatusCode>
+            </rest>
+            <setVariable name="IP_ADDRESS_ID" scope="instance"></setVariable>
+        </step>
+        <step name="returnPort" optional="false">
+            <title>Return Port</title>
+            <description>Invoke RESTful API to return port back to pool.</description>
+            <condition>
+                <expression>"1" == "1"</expression>
+                <description>Check that previous step has completed</description>
+                <targetStateSet>
+                    <description>Set state to Skipped if PORT_ID is not set.</description>
+                    <extendStateExpression>
```

Comment:
Same comment as above can be applied here to use a single state expression

Reply:
Please see previous response